src/main/java/mezz/jei/transfer/RecipeTransferUtil.java

```diff
@@ -61,6 +61,15 @@ private static IRecipeTransferError transferRecipe(RecipeTransferManager recipeT
 		return transferHandler.transferRecipe(container, recipeLayout, player, maxTransfer, doTransfer);
 	}
 
+	public static boolean allowTransfer(@Nullable IRecipeTransferError error){
```

Comment:
Please rename this method to `allowsTransfer` and fix the spacing around braces for this method to match the code style.